### ISSUE TITLE
IAM roles per function

### ISSUE THREAD

**Issue by iamtekeste** [AnomalyInnovations/serverless-stack-demo-api#35]
I would love to see a tutorial on how to do per function IAM role to added to the website.
This https://serverless.com/framework/docs/providers/aws/guide/iam/#custom-iam-roles but explained in a much better way :)

**Comment by jayair** [comment]
Good point! Thanks for brining it up. We'll look into it for a future update.
